experimental/javascript_nodejs/qnamaker-activelearning-bot/Helpers/activeLearningHelper.js

```diff
@@ -0,0 +1,97 @@
+const https = require('https');
```

Comment:
please add the standard copyright header to this file:
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.